### PR TITLE
Bug/765 archived managers wrong status

### LINKED ISSUE #765: Archived managers do not have correct status in Property Manager List
**Describe the bug**
https://github.com/codeforpdx/dwellingly-app/blob/253a96bb0e2061055cece637e4c69eabc1fd1125/src/views/PropertyManagerList/index.jsx#L25-L32

I believe the logic here should be `&&` instead of `||` on line 25

Additionally the function that is used to format the data will return a string of Invalid Date which is truthy if it cannot format the date.

It would be good to extract this logic out to a function and unit test it in isolation to ensure the correct description is returned for the possible inputs.

**To Reproduce**
1. Create an archived property manager that has a last active time stamp
2. Go to manage/managers

**Expected behavior**
Should see archived status

### PR DESCRIPTION
**Pull request by jumperck**
### What issue is this solving?
This PR is solving the problem when showing the status of the Property Manager.
It also solves a build problem due to node version specification.

### Any helpful knowledge/context for the reviewer?
Is a `npm install` necessary? no
Any special requirements to test? no

### Please make sure you've attempted to meet the following coding standards
- [x] Code builds successfully
- [x] Code has been tested and does not produce errors
- [x] Code is readable and formatted
- [x] There isn't any unnecessary commented-out code
- [x] There aren't any unnecessary commits or files changed

If you're having trouble meeting this criteria, feel free to reach out on Slack for help!

### CODE REVIEW

File: src/utils/date/index.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const formatDate = dateString => {
+  if(isValidDate(dateString))
+    return new Date(dateString + ' UTC').toLocaleString()
```
Comment (NickSchimek) [inline src/utils/date/index.js:3]: The date is being modified after its check for validity. What if the act of modification causes the date to be invalid?

There should be a contract between the backend and frontend on the date format, but if that contract is inadvertently broken then I think we would like isValidDate to return false if appending UTC to the date causes it to be invalid.

**Review by NickSchimek — COMMENTED** [review]
Thanks for the tests! I've got one comment below. Let me know what you think.

**Comment by NickSchimek** [comment]
Thanks for all the work you put into this. I think it looks good. Let's get this merged.

**Review by NickSchimek — APPROVED** [review]

File: src/utils/date/index.js (hunk 1 above)
Comment (kauereinbold) [reply src/utils/date/index.js:3]: I agree with you Nick, we might have a problem if the contract is broken for some reason.
One suggestion is to change how we expose the dates through the API. I looked on the Manager list http call and noticed that the date is being transformed on the backend. 
image
image

image

I think we just need to send the date in ISO format and leave the front end handle it.
The only contract we will follow is that the backend sends a UTC date.

What do you think?

Comment (NickSchimek) [reply src/utils/date/index.js:3]: > I think we just need to send the date in ISO format and leave the front end handle it.
> The only contract we will follow is that the backend sends a UTC date.
> 
> What do you think?

Yes, I've been wanting to do that for a long time.

We're going to be combining the frontend and backend into one app/repository. See PR #769

It doesn't do any date transformations like the old backend. And sends all dates in ISO format.

I haven't implemented anything that formats the dates on the frontend, so if someone is wanted to they could implement that.

**Pull request merged**